File: Backend/TrackrAPI/TrackrAPI/Dtos/Seguridad/ExpedienteWrapper.cs
﻿using TrackrAPI.Models;

namespace TrackrAPI.Dtos.Seguridad
{
    public class ExpedienteWrapper
    {
        public UsuarioDto paciente { get; set; }
        public ExpedienteTrackr expediente { get; set; }
        public Domicilio domicilio { get; set; }

        public IEnumerable<ExpedientePadecimientoDTO> padecimientos { get; set; }
    }
}

File: Frontend/trackr-administrador/src/app/shared/dtos/seguridad/expediente-wrapper.ts
import { Domicilio } from '@models/seguridad/domicilio';
import { ExpedientePadecimientoDTO } from '@dtos/seguridad/expediente-padecimiento-dto';
import { ExpedienteTrackR } from '@models/seguridad/expediente-trackr';
import { Usuario } from '@models/seguridad/usuario';
export class ExpedienteWrapper{
    expediente: ExpedienteTrackR;
    paciente: Usuario;
    domicilio: Domicilio;
    padecimientos: ExpedientePadecimientoDTO[];

    constructor(){
        this.expediente = new ExpedienteTrackR();
        this.paciente = new Usuario();
        this.domicilio = new Domicilio();
        this.padecimientos = [];
    }
}